### PR TITLE
Add missing translations to guide/transitions.md

### PR DESCRIPTION
**Pull request by fedekau**
https://trello.com/c/tCC9FFuR/36-guide-transitionsmd

### CODE REVIEW

File: src/v2/guide/transitions.md [hunk 1]
```diff
@@ -1167,9 +1167,9 @@ new Vue({
 {% raw %}
 <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.14.1/lodash.min.js"></script>
 <div id="list-complete-demo" class="demo">
-  <button v-on:click="shuffle">Shuffle</button>
-  <button v-on:click="add">Add</button>
-  <button v-on:click="remove">Remove</button>
+  <button v-on:click="shuffle">Mezclar</button>
+  <button v-on:click="add">Agreagar</button>
```
Comment (FLarra) [inline src/v2/guide/transitions.md]:
```suggestion
  <button v-on:click="add">Agregar</button>
```

File: src/v2/guide/transitions.md [hunk 2]
```diff
@@ -853,7 +853,7 @@ Se vé genial, ¿no?
 
 ## Transiciones Entre Componentes
 
-Las transiciones entre componentes es aún más sencillo - nisiquiera necesitamos el atributo `key`. En su lugar, sólo envolvemos un [componente dinámico](components.html#Componentes-Dinámicos):
+Las transiciones entre componentes es aún más sencillo - ni siquiera necesitamos el atributo `key`. En su lugar, sólo envolvemos un [componente dinámico](components.html#Componentes-Dinámicos):
```
Comment (FLarra) [inline src/v2/guide/transitions.md]:
```suggestion
Las transiciones entre componentes son aún más sencillas - ni siquiera necesitamos el atributo `key`. En su lugar, sólo envolvemos un [componente dinámico](components.html#Componentes-Dinámicos):
```

File: src/v2/guide/transitions.md [hunk 3]
```diff
@@ -940,8 +940,8 @@ Ahora veamos un ejemplo sencillo, transiciones de entrada y salida usando las mi
 
 ``` html
 <div id="list-demo">
-  <button v-on:click="add">Add</button>
-  <button v-on:click="remove">Remove</button>
+  <button v-on:click="add">Agragar</button>
```
Comment (FLarra) [inline src/v2/guide/transitions.md]:
```suggestion
  <button v-on:click="add">Agregar</button>
```

File: src/v2/guide/transitions.md [hunk 4]
```diff
@@ -1221,10 +1221,10 @@ Estas animaciones FLIP no están limitadas a un eje particular. Los elementos en
 
 {% raw %}
 <div id="sudoku-demo" class="demo">
-  <strong>Lazy Sudoku</strong>
-  <p>Keep hitting the shuffle button until you win.</p>
+  <strong>Sudoku Perezoso</strong>
+  <p>Continua apretando el botón mezclar hasta que ganes.</p>
```
Comment (FLarra) [inline src/v2/guide/transitions.md]:
```suggestion
  <p>Continúa apretando el botón mezclar hasta que ganes.</p>
```

File: src/v2/guide/transitions.md [hunk 5]
```diff
@@ -1498,24 +1498,24 @@ Pero en realidad, cualquier transición puede ser asignada dinámicamente. Y no
 <script src="https://cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.min.js"></script>
 
 <div id="dynamic-fade-demo" class="demo">
-  Fade In: <input type="range" v-model="fadeInDuration" min="0" v-bind:max="maxFadeDuration">
-  Fade Out: <input type="range" v-model="fadeOutDuration" min="0" v-bind:max="maxFadeDuration">
+  Fundido de Entrada: <input type="range" v-model="fadeInDuration" min="0" v-bind:max="maxFadeDuration">
```
Comment (FLarra) [inline src/v2/guide/transitions.md]:
```suggestion
  transición de Entrada: <input type="range" v-model="fadeInDuration" min="0" v-bind:max="maxFadeDuration">on
```

File: src/v2/guide/transitions.md [hunk 6]
```diff
@@ -1498,24 +1498,24 @@ Pero en realidad, cualquier transición puede ser asignada dinámicamente. Y no
...
+  Fundido de Salida: <input type="range" v-model="fadeOutDuration" min="0" v-bind:max="maxFadeDuration">
```
Comment (FLarra) [inline src/v2/guide/transitions.md]:
```suggestion
  Transición de Salida: <input type="range" v-model="fadeOutDuration" min="0" v-bind:max="maxFadeDuration">
```

File: src/v2/guide/transitions.md [hunk 7]
```diff
@@ -1569,24 +1569,24 @@ new Vue({
 {% raw %}
 <script src="https://cdnjs.cloudflare.com/ajax/libs/velocity/1.2.3/velocity.min.js"></script>
 <div id="dynamic-fade-demo" class="demo">
-  Fade In: <input type="range" v-model="fadeInDuration" min="0" v-bind:max="maxFadeDuration">
-  Fade Out: <input type="range" v-model="fadeOutDuration" min="0" v-bind:max="maxFadeDuration">
+  Fundido de Entrada: <input type="range" v-model="fadeInDuration" min="0" v-bind:max="maxFadeDuration">
```
Comment (FLarra) [inline src/v2/guide/transitions.md]: The same here? `Transición` instead of `Fundido`?

**Review by FLarra — COMMENTED** [review]
There are some typos, also I shared one suggestion. What do you think?

**Comment by fedekau** [comment]
I agree with the suggestion, just fixed the typos and the suggestion!

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**